lib/node/nodes/trade-area/trade-area.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var Node = require('../../node');
+var buildQuery = require('./trade-area-query-builder');
+var buildRange = require('./data-range-builder');
```

Comment:
I would prefer to keep it everything in here: no builders, etc.

If a node is too complex probably we have to consider splitting it into different nodes.